Engine/TransactionHandlers/BrokerageTransactionHandler.cs

```diff
@@ -267,6 +275,20 @@ public OrderTicket AddOrder(SubmitOrderRequest request)
             return ticket;
         }
 
+        /// <summary>
+        /// For backtesting we will process the order ourselves
+        /// </summary>
+        /// <param name="ticket">The <see cref="OrderTicket"/> expecting to be processed</param>
+        protected virtual void WaitForOrderToBeProcess(OrderTicket ticket)
```

Comment:
Perhaps this could be named a little better -- when I hear 'order processed' it makes me think the order has reached a completed state, but the checks below seem to imply that LEAN has done initial processing/submission of order but it might not yet be filled. Perhaps something like `WaitForOrderSubmission` -- (naming is hard!)